src/molecules/formfields/ToggleField/Readme.md

```diff
@@ -8,8 +8,8 @@
     ]}
     tooltipMessage='You can choose either of these awesome options!'
     input={{
-      value: state.value,
-      onChange: (value) => setState({ value })
+      value: 'state.value',
```

Comment:
@cisacke @patrickkim The Readme still doesn't have any variables in it -- I don't see how his fix addresses this?